configs/nextflow.base.config

```diff
@@ -1,22 +1,22 @@
 params {
```

Comment:
Please let the example config remain as is. Make a seperate config for your local system, or add it to .gitignore